src/daemon/settings.vala

```diff
@@ -285,8 +285,7 @@ public class SettingsManager {
         }
 
         this.xoverrides.set_value("overrides", xset);
-        this.wm_settings.set_string("button-layout", wm_set);
```

Comment:
Let's preserve the setting of our key as per our earlier conversation in this PR.